src/main/index.ts

```diff
@@ -24,17 +27,19 @@ app.post('/upgrade', async (req: any, res: any) => {
         const upgradeArr: Array<IUpgradeMessage> = req.body;
         const upgradeService = new UpgradeService(upgradeArr);
         const upgradeResponse = await upgradeService.upgradeDeployment();
-        if(upgradeResponse.upgradeResult == UpgradeResult.Success) {
-            res.status(200).send({
+        if(upgradeResponse.upgradeResult === UpgradeResult.Success) {
+            res.status(200).json({
                 message: `Successfuly upgraded ${upgradeResponse.upgradeCount} containers`
             });
```

Comment:
You need to return here, otherwise you'll get an error that headers are already sent. Neither status nor json nor send terminates execution.

Reply:
Done.